Release.md

```diff
@@ -0,0 +1,3 @@
+Release type: bugfix
```

Comment:
`bugfix` → `patch`

Acceptable values here are: `major`, `minor`, and `patch` — corresponding to values accepted by `poetry version` command that is automatically run by AutoPub.

Reply:
fixed!